bats_test/create-drive/drive-name-validation.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+# Support lib
+load '/home/node/packages/node_modules/bats-support/load.bash'
+# Assertions
+load '/home/node/packages/node_modules/bats-assert/load.bash'
+
+@test "create drive with empty name" {
+    run -1 bash -c "yarn ardrive create-drive --dry-run -n '' -w $WALLET"
+
+    assert_line -n 0 "Error: Required parameter driveName wasn't provided!"
+}
+
+@test "create drive with long name" {
+    run -1 bash -c "yarn ardrive create-drive --dry-run -n '+===============================================================================================================================================================================================================================================================' -w $WALLET"
```

Comment:
Might be nice to factor out this long name as a string, assert that its length is 255 characters before the test starts and then use the variable in the ardrive invocation.

Reply:
I create a constant for this string [926db0a](https://github.com/ardriveapp/ardrive-cli/pull/247/commits/926db0a540bd62fbd7070aa66f12ff6f385d002e)